code/Makefile

```
@@ -55,21 +55,21 @@ CFLAGS	+=	$(INCLUDE) -DARM11 -D_3DS $(VERFLAGS)
 
 CXXFLAGS	:= $(CFLAGS) -fno-rtti -fno-exceptions -std=gnu++11
 
+LDFLAGS	=	-g $(ARCH) -Wl,-Map,$(notdir $*.map) -T $(TOPDIR)/oot.ld $(VERFLAGS) -lgcc
```

```suggestion

LINK_SCRIPT	:= oot.ld

LDFLAGS	=	-g $(ARCH) -Wl,-Map,$(notdir $*.map) -T $(TOPDIR)/$(LINK_SCRIPT) $(VERFLAGS) -lgcc
```
You shouldn't delete the `LINK_SCRIPT` variable because the `.elf` target depends on it.
This is needed to make `make` detect changes on the linker script even if it's the only modified file.